dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -353,14 +353,14 @@
 
             // refresh MethodConfigs
             List<MethodConfig> methodConfigs = this.getMethods();
-            if (methodConfigs != null && methodConfigs.size() > 0) {
+            if (methodConfigs != null && !methodConfigs.isEmpty()) {
```

Comment:
**Suggestion**: Replace size() > 0 check with !isEmpty() for clarity and performance. <br> **Code Suggestion**: 
 ```
 +            if (methodConfigs != null && !methodConfigs.isEmpty()) {
 ```